### PR TITLE
refactor: cleanup storage-proofs

### PR DESCRIPTION
**Pull request by dignifiedquire**
This only moves files arounds, with the goal to cleanup some imports and give a clearer structure of what belongs where. First in multiple steps to clean up an improve the state of the code base

### CODE REVIEW

File: storage-proofs/src/porep/drg/compound.rs [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+/// * `replica_parents` - A list of all parents in the replica, with their value.
+/// * `replica_parents_paths` - A list of all parents paths in the replica.
+///
+/// ----> Private `data_node` - The data node being proven.
+///
+/// * `data_node_path` - The path of the data node being proven.
+/// * `data_root` - The merkle root of the data.
+/// * `replica_id` - The id of the replica.
+///
+
```
Comment (DrPeterVanNostrand) [inline storage-proofs/src/porep/drg/compound.rs]: Blank line

File: storage-proofs/src/porep/drg/circuit.rs [hunk 2]
```diff
@@ -46,29 +40,23 @@ use crate::util::bytes_into_boolean_vec_be;
...
-//    DrgPoRepCircuit,
-//    DrgPoRepCompound,
-//    "drg-proof-of-replication",
-//    false
-//);
 
 pub struct DrgPoRepCircuit<'a, H: Hasher> {
-    params: &'a <Bls12 as JubjubEngine>::Params,
-    replica_nodes: Vec<Option<Fr>>,
+    pub params: &'a <Bls12 as JubjubEngine>::Params,
```
Comment (DrPeterVanNostrand) [inline storage-proofs/src/porep/drg/circuit.rs]: Do these fields need to be pub?

File: storage-proofs/src/porep/drg/compound.rs (hunk 1 above)
Comment (dignifiedquire) [reply storage-proofs/src/porep/drg/compound.rs]: 💥

File: storage-proofs/src/porep/drg/circuit.rs (hunk 2 above)
Comment (dignifiedquire) [reply storage-proofs/src/porep/drg/circuit.rs]: brings them in line with the rest, the technically only need to be `pub(crate)`

**Pull request merged**